channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -283,6 +288,11 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControls {
                         }
                     }
 
+                    if (!config.disableStatsBeat && _statsBeat && !_statsBeat.isInitialized()) {
```

Comment:
You need to "set" a default otherwise `disableStatsBeat` will be undefined and not dynamic.

I'm also in 2 minds on this as we currently don't have any throttling in place, so once someone turns this one (by looking at the code or the tests), this will spam (overload) our statsbeat subscription with no option to stop it....

So we "might" want to check this code in with all of the "enable" commented out so there is zero chance of someone accidently enabling this. And then as part of the follow up PR to add the throttling we then add the config and support for "enabling" / "disabling".

Basically, we don't want someone turning this on a killing ALL of our statsbeat metrics. As the end-user (customers) subscription won't see any of this data.